### DISCUSSION TITLE
Alias/rename Provider to WagmiProvider

### DISCUSSION THREAD

**Discussion by markdalgleish** [wevm/wagmi#158] [Ideas]
If you use any other provider components in your app (which is very likely) you're forced to alias the `Provider` import yourself, both to avoid collisions and to improve code clarity. It would be nice if the provider already had a name designed for this:

```diff
-import { Provider as WagmiProvider } from 'wagmi';
+import { WagmiProvider } from 'wagmi';
```

This could be done in a non-breaking way by exposing the original `Provider` export as well as the `WagmiProvider` alias (and `WagmiProviderProps`). The only downside I can see is that it makes documentation a bit more difficult—but not too bad.

**Comment by tmm** [comment]
Let's do it!

**Reply by markdalgleish** [reply]
Great! I'm happy to PR this if you like.

One question: would you want to change the canonical name to `WagmiProvider` in the docs and remove references to `Provider`? Or would you want to keep the docs referencing `Provider` but as a side note present `WagmiProvider` as an alias?

**Reply by tmm** [reply]
Let's do the latter. Makes it look cleaner, but presents the other option.

**Reply by markdalgleish** [reply]
PR: https://github.com/tmm/wagmi/pull/159
